src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -124,6 +126,16 @@ def which(binary):
     return None
 
 
+def get_cmd_test_hook_data(filename):
+    hook_data = ""
```

Comment:
if empty, return one None.

Reply:
Sure, changed in commit [0f6de86](https://github.com/Azure/azure-cli/pull/19115/commits/0f6de86183629cd7eb5e358af52cbf4ee8a17016).